apps/admin/backend/src/reports/ballot_count_report.ts

```diff
@@ -88,14 +88,19 @@ export async function generateBallotCountReportPreview({
 
   ...reportProps
 }: BallotCountReportPreviewProps): Promise<BallotCountReportPreview> {
-  const report = buildBallotCountReport(reportProps);
   await logger.logAsCurrentRole(LogEventId.ElectionReportPreviewed, {
     message: `User previewed a ballot count report.`,
     disposition: 'success',
```

Comment:
is it possible to make this log as a failure if there is an error in getting the report preview?

Reply:
good call, thanks 40d199cfa